COMPILING.md

```diff
@@ -314,6 +314,10 @@ If compiling with cmake:
    cmake --build build
    ```
 
+If compiling with cmake Ninja generator (`cmake -G Ninja`), the extra step
+`cmake --build build --target cudd-make` may be necessary between step 1 and 2
+because of a problem in the tracking of dependencies.
```

Comment:
Is there a issue raised about the problem with tracking dependencies? If so, could you link to it in the docs here. If not, could I be cheeky and ask you to create that issue as well :-) @hannes-steffenhagen-diffblue is experimenting with some cleanups/rewrites of the CMake build files - so having all the known problems tracked as issues would give him a fighting chance of fixing some of these at the same time.